pytorch_lightning/trainer/trainer.py

```diff
@@ -169,7 +169,9 @@ def __init__(
 
             callbacks: Add a list of callbacks.
 
-            checkpoint_callback: Callback for checkpointing.
+            checkpoint_callback: If ``True``, will configure a default ModelCheckpoint callback.
+                Passing a ModelCheckpoint instance to this argument is deprecated since
+                v1.1.0 and will be unsupported from v1.4.0. Default: ``True``
```

Comment:
**If ``True``, will configure a default ModelCheckpoint callback.**
So to pass custom ModelCheckpoint, it needs to be false?
But, answer is no, right?
Just to be sure for this case (appeared in mind when I first read it)
```suggestion            
            checkpoint_callback: If ``True``, enable checkpointing.
                It will configure a default ModelCheckpoint callback
                if there is no custom ModelCheckpoint callback in `:paramref:~Trainer.callbacks`.
                Default: ``True``.

                .. warning:: Passing a ModelCheckpoint instance to this argument is deprecated since
                    v1.1.0 and will be unsupported from v1.4.0.
```